aws/resource_aws_dynamodb_table.go

```diff
@@ -418,6 +430,12 @@ func resourceAwsDynamoDbTableCreate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
+	if _, ok := d.GetOk("replica"); ok {
+		if err := createDynamoDbReplicas(d.Id(), d.Get("replica").([]interface{}), conn); err != nil {
```

Comment:
To go with our attribute change to `TypeSet`

```suggestion
		if err := createDynamoDbReplicas(d.Id(), d.Get("replica").(*schema.Set).List(), conn); err != nil {
```